packages/headless/src/controllers/facets/range-facet/date-facet/date-range.ts

```diff
@@ -1,13 +1,13 @@
 import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
-import customParseFormat from 'dayjs/plugin/customParseFormat';
-import {DateRangeRequest} from '../../../../features/facets/range-facets/date-facet-set/interfaces/request';
 import {FacetValueState} from '../../../../features/facets/facet-api/value';
+import {formatDateForSearchApi} from '../../../../api/date-format';
+import {RelativeDate} from '../../../../features/relative-date-set/relative-date';
+import {DateRangeRequest} from '../../../../features/facets/range-facets/date-facet-set/interfaces/request';
 
 dayjs.extend(utc);
-dayjs.extend(customParseFormat);
 
-export type DateRangeInput = string | number | Date;
+export type DateRangeInput = string | number | Date | RelativeDate;
```

Comment:
Modified this module in order to be able to pass a RelativeDate as well as another type of range.
Previously in my other branch we had a new buildRelativeRange module instead. But now mixed ranges are supported (start is absolute, end is relative)